lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyCompletionParticipant.java

```diff
@@ -67,7 +79,7 @@ public void onAttributeValue(String valuePrefix, ICompletionRequest request, ICo
                     completionItem.setTextEdit(edit);
                     completionItem.setFilterText(variableProp.getKey().toString());
                     completionItem.setKind(CompletionItemKind.Value);
-                    completionItem.setDocumentation(String.format("%s = %s", variableProp.getKey(),variableProp.getValue()));
+                    completionItem.setDocumentation(String.format("%s = %s", variableProp.getKey(), variableProp.getValue()));
```

Comment:
Is this showing the value during completion? I thought we decided against that.

Reply:
It is showing only in documentation section of completion, not in actual value
<img width="854" alt="image" src="https://github.com/user-attachments/assets/a2013238-8718-470e-949b-04af48cc628c" />